client/my-sites/media-library/test/data-source.jsx

```diff
@@ -53,8 +54,8 @@ describe( 'MediaLibraryDataSource', () => {
 					/>
 				</ReduxProvider>
 			);
-			expect( wrapper.find( 'button[data-source="google_photos"]' ) ).toHaveLength( 1 );
-			expect( wrapper.find( 'button[data-source="pexels"]' ) ).toHaveLength( 0 );
+			expect( screen.getByText( 'Google Photos' ) ).toBeInTheDocument();
+			expect( screen.queryByText( 'Pexels free photos' ) ).toBe( null );
```

Comment:
What about using `queryByText` for both these.

Reply:
updated it